java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -727,19 +727,51 @@ class DatabaseImpl(
         writableDatabase.transaction {
             // Find all expired entities.
             val storageKeyIds = rawQuery(
```

Comment:
Maybe rename this to better indicate what it contains? storageKeyPairs, perhaps?

Reply:
Even better might be to pull it apart into two separate lists: storageKeyIds and storageKeys